components/App/App.tsx

```diff
@@ -3,10 +3,17 @@ import type LoadActivity from '../../lib/LoadActivity.js';
 import GraphDiagram from '../GraphDiagram/GraphDiagram.js';
 import Inspector from '../Inspector.js';
 import './App.scss';
+import { useQuery } from '../../lib/useQuery.js';
+import Intro from '../Intro.js';
 import { Loader } from './Loader.js';
 
 export default function App() {
   const activity = useActivity();
```

Comment:
@foxt nice catch! I think it's a matter of moving `useExternalInput` from `Inspector.tsx` to this line here. PR welcome